src/index.ts

```diff
@@ -28,6 +28,7 @@ function run(updateDetails?: IUpdateDetails) {
         --clean, -c             Clean generated output folders at the beginning
         --threads, -t           Maximum number of threads to run in parallel
         --localonly, -l         Enable to not scan other directories than CWD for plugins
+        --noparents, -x         Enable to only run compilation on plugins in current repository (still scans for other sources to be present)
```

Comment:
As far as I understand, `--localonly` does a full compilation, but operates on a different folder structure (everyhting in a single directory). I don't know when we need this, but it's kind of easy to confuse with `--noparents`.